charts/risingwave/values.yaml

```
@@ -971,6 +971,13 @@ computeComponent:
   ##
   replicas: 1
 
+  ## @param computeComponent.autoscaling Compute Pod horizontal Pod Autoscalers
```

Add a reference to the details of the autoscaler object.

```suggestion
  ## @param computeComponent.autoscaling Compute Pod horizontal Pod Autoscalers
  ## Reference: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#container-resource-metrics
  ##
```

---

fixed.